assets/js/modules/subscribe-with-google/components/posts/AccessSelector.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Subscribe with Google Access Selector component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../datastore/constants';
+const { useSelect } = Data;
+
+export default function AccessSelector( { hidden, onChange, selectedOption } ) {
```

Comment:
So far I don't think this component actually needs `onChange` and `selectedOption` props. It would simplify the code to fully internalize that into this component, basically rely on `useState` to capture the current option and then implement an internal `onChange` that sets the selected option in state.